modules/p2/ding_serendipity/ding_serendipity.module

```diff
@@ -221,8 +220,11 @@ function ding_serendipity_fetch(array $context = NULL) {
         $cache_id[] = $ct;
       }
     }
-    $cache_id = "ding_serendipity" . implode(':', $cache_id);
 
+    // Prefix cid with uid for per-user serendipity cache clear.
+    $user_key = isset($context['current-user']) ? ("_" . $context['current-user']) . "_" : "";
```

Comment:
Nitpicking: What is going on with the placement of `()` in the first branch of the ternary statement? It does not seem like they are needed at all.